pages/testpage.tsx

```diff
@@ -1,13 +1,17 @@
-﻿import { NextPage } from 'next';
-import Layout from "../components/Layout/layout";
-import scss from "../pageStyles/index.module.scss";
+﻿import {NextPage} from 'next';
 import React from "react";
+import Layout from "../components/Layout/layout";
+import CandidateTestView from "../components/CandidateTestView/CandidateTestView";
+import {TestList} from "../components/CandidateTestView/Tests/TestList";
 
 const TestPage: NextPage = () => {
+    const testToRender = TestList[0];
```

Comment:
can we make this change what test it renders based on the URL or something?

Reply:
That's the plan :) index should correspond to test number. Do we need to add this functionality for MVP with just one test? Changed it to show Sasha's test

Reply:
MVP will have 3 tests - so yep! :)

Reply:
Ok, can we merge for now and open another ticket to work on that?

Reply:
I don't mind doing it as a separate PR - that seems like a good plan. :) 
but I think its the same ticket - because its fairly core to the functionality.

Reply:
P.S. Sasha only sent 1 test, with a selection of different example inputs/outputs. Because we won't be running the code, the only difference these will make is a tiny bit of text, 90% of the test will look identical. Should I pull another test off hackerrank (as i did for test 3) or ask Sasha for some more?

Reply:
Perfect, very happy to continue working on it but desperate to merge this now! 😊

Reply:
yep - sasha only sent through a single test as an example.
We should ask her for more tests